ayl.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: Welcome to Engineers For Exploration!
+---
+
+Hope you are enjoying/enjoyed the info session!
+
+**The application to apply can be found at [{{ "join" | absolute_url }}]({{ "join" | relative_url }})**
+
+If you are interested in the projects, please check out the following contacts.
+
+##  [Aqua3D]({{ "fishsense" | relative_url }})
+- Christopher Crutchfield, ccrutchf@ucsd.edu
+- Avik Ghosh, a8ghosh@ucsd.edu 
+
+##  [FishSense Core Technologies]({{ "fishsense" | relative_url }})
+- Benjamin McCalmon, bbmcca26@g.holycross.edu
+
+## [FishSense End User]({{ "fishsense" | relative_url }})
+- Harish Vasanth, havasanth@ucsd.edu
+
+## [Smartfin]({{ "smartfin" | relative_url }})
+- Nathan Hui, nthui@ucsd.edu
+
+## [Radio Telemetry Tracking]({{ "radio-collar-tracker" | relative_url }})
+- Tyler Flar (Radio Telemetry Tracking) – tflar@ucsd.edu
+
+## [Radio Telemetry Tracking]({{ "acoustic-species-identification" | relative_url }}) 
+- Sean Perry, shperry@ucsd.edu
+- Ludwig von Schoenfeldt, lvonschoenfeldt@ucsd.edu
+- TQ Zhang, tiz019@ucsd.edu 
+
+##  [Mangrove Monitoring]({{ "mangrove-monitoring" | relative_url }}) 
+- Gage Wyre, gwrye100@gmail.com 
+- Zach Lawrence, zclawrence@ucsd.edu
+
+## [Hardware Dev/Research Support]({{ "research-support-group" | relative_url }})   
+- Nathan Hui, nthui@ucsd.edu
```

Comment:
Let's drop this, I've removed it as an option from the application, and I don't really have time to manage it.